src/pkgmt/versioneer.py

```diff
@@ -113,6 +171,14 @@ def version(
             f"\n{pending.decode()}"
         )
 
+    cfg = load()
```

Comment:
You mean from `cli.py` and `tasks.py`? What if user passes `tag` and `push` as command line arguments? which one to give preference?

Reply:
we can leave tasks.py as is, but we should modify cli.py. command line arguments have preference, that's standard in any CLI tool. we can show a message to the user in case there is both a pyproject.toml setting and a CLI, to let the know that we'll override the pyproject.toml value (if it's different) with the command line argument

Reply:
moved logic to `cli.py` and added tests. Also created a new class `PyprojectConfig` in `config.py`.